lib/galaxy/webapps/galaxy/api/__init__.py

```diff
@@ -306,6 +307,16 @@ def get_current_history_from_session(galaxy_session: Optional[model.GalaxySessio
     return None
 
 
+def fix_url_for(mapper, galaxy_request: GalaxyASGIRequest):
```

Comment:
```suggestion
def fix_url_for(mapper: Mapper, galaxy_request: GalaxyASGIRequest) -> None:
```

Reply:
Is this really worth it ? This is legacy stuff we can remove once all relevant routes are FastAPI routes. If we do this I suppose we also need to annotate app.legacy_mapper, which would require typing app more precisely in get_trans.

Reply:
It seems to not be an error in mypy, though that then makes me wonder what the point of doing this is in the first place 😆

Reply:
Ah, that's because `getattr(..., ..., None)` returns `Any | None`.
I guess this is still good documentation though.